src/Endpoints/FirewallSettings.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Cloudflare\API\Endpoints;
+
+use Cloudflare\API\Adapter\Adapter;
+
+class FirewallSettings implements API
+{
+    private $adapter;
+
+    public function __construct(Adapter $adapter)
+    {
+        $this->adapter = $adapter;
+    }
+
+    /**
+     * Get the Security Level feature for a zone.
+     *
+     * @param string $zoneID The ID of the zone
+     * @return string|false
+     */
+    public function getSecurityLevelSetting(string $zoneID)
+    {
+        $return = $this->adapter->get(
+            'zones/' . $zoneID . '/settings/security_level'
+        );
+        $body = json_decode($return->getBody());
+        if (isset($body->result)) {
```

Comment:
As the package requires `php-7.0` version at least, try considering following code snippets:

```php
return $body->result->value ?? false;
```